<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configure Audio Devices</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="input_device_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Microphone:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select Virtual Audio Cable: (&lt;a href=&quot;https://vb-audio.com/Cable/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;What's this?&lt;/span&gt;&lt;/a&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="output_device_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <action name="action_configure_audio_devices">
   <property name="text">
    <string>Configure Audio Devices</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
